templates/gRPC-C++.podspec.template

```diff
@@ -136,7 +136,9 @@
       'USER_HEADER_SEARCH_PATHS' => '"$(PODS_TARGET_SRCROOT)"'${"\\"}
           ' "$(PODS_TARGET_SRCROOT)/src/core/ext/upb-generated"'${"\\"}
           ' "$(PODS_TARGET_SRCROOT)/src/core/ext/upbdefs-generated"'${"\\"}
-          ' "$(PODS_TARGET_SRCROOT)/third_party/**"',
+          ' "$(PODS_TARGET_SRCROOT)/third_party/re2"'${"\\"}
+          ' "$(PODS_TARGET_SRCROOT)/third_party/upb"'${"\\"}
+          ' "$(PODS_TARGET_SRCROOT)/third_party/xxhash"',
```

Comment:
There are more than 20 directories under third_party, I wonder how did you decide only these three are needed?

Reply:
We used to manage a list of dependencies to third_party and but it changed by https://github.com/grpc/grpc/pull/27609 so I used it to find these dependencies.

Reply:
I see, thanks for the change! LGTM!